.github/workflows/windows.yml

```diff
@@ -26,9 +26,8 @@ jobs:
         shared: ["ON", "OFF"]
         include:
           - os: "windows-2022"
-            generator: "Visual Studio 17 2022"
+            generator: "Visual Studio 16 2019"
```

Comment:
We used to check VS 17 2022 on Windows 2022 before I disabled to 2019 VMs, so this kind of goes backwards. The 2025 entry below is a guess which happened to work in the CI.

Reply:
Yeah, I wanted to check if the older compiler was at least still available on that platform since that's still supported by MS. I have VS 16 and VD 17 installed locally on my test VMs. They're independent of the OS you run them on. There's no newer VS version than VS 17, so it's kind of redundant otherwise to run this on 2 different OSes. We can probably just choose whether to CI build them all on windows-2022 or 2025.

Reply:
Yeah, looks like they just don't install the older toolchain in parallel on the newer Github CI images. I'll back this out.